Revert Facebook Pixel

librarianmage opened the pull request:
[//]: # (Note: See http://habitica.wikia.com/wiki/Using_Habitica_Git#Pull_Request for more info)

[//]: # (Put Issue # or URL here, if applicable. This will automatically close the issue if your PR is merged in)
Fixes #8390 

### Changes
[//]: # (Describe the changes that were made in detail here. Include pictures if necessary)

@SabreCat I have a second attempt to revert the files from the commit mentioned in my last PR

[//]: # (Put User ID in here - found in Settings -> API)

----
UUID:

librarianmage committed:
attempt revert

librarianmage commented:
![Ping!](http://i1178.photobucket.com/albums/x371/spectacledotter/Reaction%20Faces/stitch-book-throwing.gif)

Ping!

paglias commented:
LGTM, sorry for the delay

SabreCat commented on website/client-old/js/services/analyticsServices.js:
```diff
@@ -127,10 +107,6 @@
     }
-
-    properties.balance = user.balance;
-    properties.balanceGemAmount = properties.balance * 4;
```
Why were these removed? @MathWhiz @paglias

paglias replied:
Uh-oh I hadn't noticed that

librarianmage commented:
I just reverted the commit, and moved the files into the new structure, so
I have no idea.
On Mon, Jan 23, 2017 at 9:36 AM Sabe Jones <notifications@github.com> wrote:

> *@SabreCat* commented on this pull request.
> ------------------------------
>
> In website/client-old/js/services/analyticsServices.js
> <https://github.com/HabitRPG/habitica/pull/8406#pullrequestreview-17949455>
> :
>
> > @@ -127,10 +107,6 @@
>        properties.Level = user.stats.lvl;
>        properties.Mana = Math.floor(user.stats.mp);
>      }
> -
> -    properties.balance = user.balance;
> -    properties.balanceGemAmount = properties.balance * 4;
>
> Why were these removed? @MathWhiz <https://github.com/MathWhiz> @paglias
> <https://github.com/paglias>
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/HabitRPG/habitica/pull/8406#pullrequestreview-17949455>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALz_81BZbzWT03jIaTAw8qbzww7R6uGHks5rVMkEgaJpZM4LjQaJ>
> .
>

SabreCat commented:
Yeah, looks like in the process you grabbed and reverted some unrelated changes. Third try? :cat::sweat_drops:

paglias commented:
I've reverted this PR. @MathWhiz probably the commit included some other unrelated changes
